features/enterprise/impl/build.gradle.kts

```diff
@@ -21,6 +21,7 @@ dependencies {
     api(projects.features.enterprise.api)
     implementation(projects.libraries.architecture)
     implementation(projects.libraries.matrix.api)
+    api(libs.compound)
```

Comment:
Are these `api` needed (here and in the api module)? Couldn't they just be `implementation` since the designsystem module will already provide the compound dependency?

Reply:
You're right: https://github.com/element-hq/element-x-android/pull/4226/commits/b7dc2e3551132a8dc06acfd525d008013909ca09